### DISCUSSION TITLE
Black map on iPhone simulator 8 with iOS 16

### DISCUSSION THREAD

**Discussion by SERCHAT** [rnmapbox/maps#3546] [Q&A] [answered]
I'm getting a black map error in the iPhone 8 simulator. I made all the configurations as explained in the [link ](https://github.com/rnmapbox/maps/blob/main/ios/install.md)I mentioned. It works properly on other iPhone models and iOS models, but I only see a black screen on the iOS 16 iPhone 8 simulator, what could be the reason?

**Comment by mfazekas** [comment, marked as answer]
@SERCHAT not sure, pls check the console in Xcode, mapbox might have logged something there.

Please also check https://rnmapbox.github.io/docs/install

**Reply by VicHofs** [reply]
@SERCHAT could you please share what the fix was for you?

**Reply by mfazekas** [reply]
See also this https://github.com/rnmapbox/maps/issues/3824 should have been addressed with 11.12.0
